Author: Luciano Bello <luciano@linux.org.ar>
Description: avoid the "implicit declaration of function 'ntohs'" warning
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/decode_aim.c
+++ b/decode_aim.c
@@ -14,6 +14,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "hex.h"
 #include "buf.h"
--- a/decode_mmxp.c
+++ b/decode_mmxp.c
@@ -21,6 +21,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "buf.h"
 #include "decode.h"
--- a/decode_pptp.c
+++ b/decode_pptp.c
@@ -16,6 +16,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "buf.h"
 #include "decode.h"
--- a/decode_tds.c
+++ b/decode_tds.c
@@ -19,6 +19,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <strlcat.h>
+#include <arpa/inet.h>
 
 #include "decode.h"
 
--- a/decode_vrrp.c
+++ b/decode_vrrp.c
@@ -15,6 +15,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <arpa/inet.h>
 
 #include "buf.h"
 #include "decode.h"
--- a/ssh.c
+++ b/ssh.c
@@ -23,6 +23,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <arpa/inet.h>
 #include <unistd.h>
 
 #include "hex.h"
